prototype/app/views/question-responses.html

```diff
@@ -220,20 +220,23 @@ <h2 class="govuk-heading-m">Filters</h2>
         <tr class="govuk-table__row">
           <th scope="col" class="govuk-table__header">Closed response</th>
           <th scope="col" class="govuk-table__header">Open response</th>
-          <th scope="col" class="govuk-table__header" style="width: 10.25rem;">Actions</th>
+          <th scope="col" class="govuk-table__header" style="width: 9rem;">Actions</th>
         </tr>
       </thead>
       <tbody class="govuk-table__body">
         {% for question in data.questions %}
           <tr class="govuk-table__row">
-            <td class="govuk-table__cell">Yes</td>
+            <td class="govuk-table__cell">Agree</td>
             <td class="govuk-table__cell">{{question}}</td>
             <td class="govuk-table__cell">
+              <a href="/respondent">View respondent</a>
```

Comment:
How do we feel about "respondent" vs, say "response" here? My instinct is that the `response` is the thing we'd view, and the `respondent` would be a feature of the `response`